setup.py

```diff
@@ -63,7 +63,7 @@ def check_setuptools_features():
     # TODO Consider not installing the db drivers, or putting them in extras.
     'rethinkdb~=2.3',  # i.e. a version between 2.3 and 3.0
     'pymongo~=3.4',
-    'pysha3==1.0.0',
+    'pysha3==1.0.2',
```

Comment:
From [Python Packaging User Guide](https://packaging.python.org/requirements/#install-requires):

> It is not considered best practice to use install_requires to pin dependencies to specific versions, or to specify sub-dependencies (i.e. dependencies of your dependencies). This is overly-restrictive, and prevents the user from gaining the benefit of dependency upgrades.

We could use the `~` instead:

```python
pysha3~=1.0`
```

or 

```python
pysha3~=1.0.2`
```